test/dnsimple/certificates.spec.js

```diff
@@ -315,8 +316,8 @@ describe('certificates', function () {
 
       dnsimple.certificates.issueLetsencryptCertificateRenewal(accountId, domainId, certificateId, certificateRenewalId)
         .then(function (response) {
-          var certificateRenewal = response.data;
-          expect(certificateRenewal.id).to.eq(certificateRenewalId);
```

Comment:
the names here was a little bit different of how issueRenewal response works. We return the new certificate directly instead of the renewal object.

Reply:
Good catch 👍